ios/Video/RCTVideo.m

```diff
@@ -394,6 +415,11 @@ - (void)setSrc:(NSDictionary *)source
       [self->_player addObserver:self forKeyPath:externalPlaybackActive options:0 context:nil];
       self->_isExternalPlaybackActiveObserverRegistered = YES;
       
+      // Create IMA Ads loader and content playhead
+      self.contentPlayhead = [[IMAAVPlayerContentPlayhead alloc] initWithAVPlayer:self->_player];
```

Comment:
```suggestion
      self.contentPlayhead = [[IMAAVPlayerContentPlayhead alloc] initWithAVPlayer:_player];
```
I'm not 100% sure about this but I thought this would work as well since the _ already indicates that it is in the self class